make.bat

```diff
@@ -151,7 +156,18 @@ START cover\index.html
 GOTO end
 
 :clean
-DEL pyvxl\*.pyc 2>NUL
+SLEEP 0.2

```

Comment:
The sleep command is only available if the user has installed a Windows Resource Kit (e.g. [https://www.microsoft.com/en-us/download/details.aspx?id=17657](https://www.microsoft.com/en-us/download/details.aspx?id=17657)) or otherwise installed the sleep command.

If the delay is important, you may not want to rely on the sleep command being present.

Reply:
Interesting, I was wondering why it didn't cause you an issue earlier. I'll update it to use python since it should be installed at this point.